medcat/cat.py

```diff
@@ -184,15 +185,14 @@ def load_model_pack(cls, zip_path):
 
         return cls(cdb=cdb, config=cdb.config, vocab=vocab, meta_cats=meta_cats)
 
-
-    def __call__(self, text: str, do_train: bool = False):
+    def __call__(self, text: Optional[str], do_train: bool = False) -> Optional[Union[Doc, List[Doc]]]:
```

Comment:
I think here you are saying text has to be a string, but also saying it can return a List[Doc]. I think it should only accept one str and return Doc

Reply:
Yes, I remember we reached that agreement on single string only. Done.